packages/editor/src/Sx/ThemeColorPicker.tsx

```diff
@@ -1,16 +1,28 @@
 /** @jsx jsx */
-import { jsx, useThemeUI } from 'theme-ui'
+import { jsx, useThemeUI, Theme } from 'theme-ui'
 import React from 'react'
 import tinycolor from 'tinycolor2'
-import { GithubPicker } from 'react-color'
+import { GithubPicker, ColorState } from 'react-color'
 import { usePopoverState, Popover, PopoverDisclosure } from 'reakit/Popover'
 
-export const ThemeColorPicker = ({ children, theme, ...props }) => {
+type ThemeColorPickerProps = {
+  children?: React.ReactNode
+  theme?: Theme
+  value?: string
+  onChange: (color: string | ColorState) => void
+}
+
+export const ThemeColorPicker = ({
+  children,
+  theme,
+  ...props
+}: ThemeColorPickerProps) => {
   const popover = usePopoverState()
   const context = useThemeUI()
   // todo: look into supporting v0.2 functionality
   // const { colors } = theme || context.theme || {}
   const _theme = theme || context.theme || {}
+  // FIXME: Where to add rawColors type?
   const colors = _theme.rawColors || _theme.colors || {}
```

Comment:
`Property 'rawColors' does not exist on type 'Theme'.ts(2339)`

I'm not sure where `rawColors` should get defined, a search through the codebase doesn't seem to bring any results. Probably on the `Theme` type directly, but not sure.

Reply:
@jxnblk What's `rawColors`?

Reply:
Can't remember if this was removed, but I think this *might* be a remnant from the previous version -- doing a code search looks like it might be some dead code that was missed...

Reply:
I’ll remove it in this PR then. 👍